### PR TITLE
Fixpackage: pyxb

### PR DESCRIPTION
**Pull request by rhimmelbauer**
Notes:

- Added migration to add the `ved` currency.
- Updated setup to move fwd to using the latest Django and Python versions. 
- Added CustomDate Class that overrides the Date class form pyxb package for Authorize.Net Payment Processor.

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
     include_package_data=True,
     python_requires=">=3.6",
     install_requires=[
-        'Django>=3.1,<3.2',
+        'Django>=3.2,<4.1',
```
Comment (jared-hardy) [inline setup.py:48]: Shouldn't this be:
```
        'Django>=3.2,<=4.1',
```
?

**Review by jared-hardy — APPROVED** [review]
Looks good so far. I'd have to do a full regex on the repo to be sure every deprecation was caught. I kinda wish they'd publish a regex to search for every deprecation along with release notes.

File: setup.py (hunk 1 above)
Comment (renderbox) [inline setup.py:48]: Does it work with Django 4.0?

File: develop/core/models.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from django.utils.translation import ugettext as _
+from django.utils.translation import gettext as _
```
Comment (renderbox) [inline develop/core/models.py:1]: Does this work in Django 3.x?

File: vendor/processors/authorizenet.py [hunk 3]
```diff
@@ -29,6 +29,19 @@
 from .base import PaymentProcessorBase
 
 
+
+class customdate(pyxb.binding.datatypes.date):
```
Comment (renderbox) [inline vendor/processors/authorizenet.py:33]: Classes are typically Camel Case.  Any issue with this being lowercase?

File: setup.py (hunk 1 above)
Comment (rhimmelbauer) [reply setup.py:48]: Yes, actually that is the only thing I have been testing it with and 3.2

File: develop/core/models.py (hunk 2 above)
Comment (rhimmelbauer) [reply develop/core/models.py:1]: It works with 3.2

File: vendor/processors/authorizenet.py (hunk 3 above)
Comment (rhimmelbauer) [reply vendor/processors/authorizenet.py:33]: Agree will change it to Camel Case

**Pull request merged**